### ISSUE TITLE
feat: `*_classification` files gain column indicating the `version` or `revision` of the bridge, if relevant

### ISSUE THREAD

**Issue by jdhoffa** [RMI-PACTA/r2dii.data#344]
Some classification systems (e.g. ISIC) have numerous different versions of the classification system. 
It would be good to track this in a user-facing in, for example, the individual `*_classification` datasets, while recognizing that that column would never actually be used in the code. 

My proposal would be that:
* the `*_classification` datasets gain a column indicating this information if necessary
* the `sector_classifications` dataset keeps only the crucial columns used by the code (e.g. `sector`, `borderline`, `code`, `code_system`)

cc @jacobvjk 

[AB#10174](https://dev.azure.com/RMI-PACTA/d77fc8f8-abac-42ae-9e31-846f9901290d/_workitems/edit/10174)

**Comment by jdhoffa** [comment]
Update:

- [ ] nace_classification
- [ ] naics_classification
- [ ] sic_classification
- [ ] gics_classification
- [ ] psic_classification
- [ ] isic_classification

``` r
library(r2dii.data)

classifications <- list(
  nace_classification = nace_classification,
  naics_classification = naics_classification,
  sic_classification = sic_classification,
  gics_classification = gics_classification,
  psic_classification = psic_classification,
  isic_classification = isic_classification
)

lapply(classifications,names)
#> $nace_classification
#> [1] "original_code" "code"          "code_level"    "sector"       
#> [5] "borderline"   
#> 
#> $naics_classification
#> [1] "code"        "naics_title" "sector"      "borderline" 
#> 
#> $sic_classification
#> [1] "code"        "description" "sector"      "borderline" 
#> 
#> $gics_classification
#> [1] "code"        "code_level"  "sector"      "borderline"  "description"
#> 
#> $psic_classification
#> [1] "original_code" "code_level"    "code"          "sector"       
#> [5] "borderline"   
#> 
#> $isic_classification
#> [1] "ISIC Rev 5 Code"  "ISIC Rev 5 Title" "code"             "sector"          
#> [5] "borderline"       "revision"
```

<sup>Created on 2024-03-12 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

**Comment by jdhoffa** [comment]
Closed by #357
